js/bitmex.js

```diff
@@ -2240,22 +2430,34 @@ module.exports = class bitmex extends Exchange {
         [ tag, params ] = this.handleWithdrawTagAndParams (tag, params);
         this.checkAddress (address);
         await this.loadMarkets ();
-        // let currency = this.currency (code);
-        if (code !== 'BTC') {
-            throw new ExchangeError (this.id + ' supoprts BTC withdrawals only, other currencies coming soon...');
-        }
         const currency = this.currency (code);
+        const precision = this.safeString (currency, 'precision');
+        const amountString = this.numberToString (amount);
+        const amountFinal = Precise.stringMul (amountString, precision);
+        const networkCode = this.safeString (params, 'network');
+        const networkId = this.networkCodeToId (networkCode);
         const request = {
-            'currency': 'XBt', // temporarily
-            'amount': amount,
+            'currency': currency['info']['currency'], // this is specific currency-slug, like XBt, which differs from currency['id'] XBT
+            'amount': amountFinal,
             'address': address,
+            'network': networkId,
             // 'otpToken': '123456', // requires if two-factor auth (OTP) is enabled
             // 'fee': 0.001, // bitcoin network fee
         };
         const response = await this.privatePostUserRequestWithdrawal (this.extend (request, params));
         return this.parseTransaction (response, currency);
     }
 
+    networkCodeToId (networkCode) {
```

Comment:
@ttodua we can remove these from there here right?

Reply:
right.